tools/velvet/velveth.xml

```diff
@@ -1,70 +1,97 @@
 <tool id="velveth" name="velveth" version="@WRAPPER_VERSION@.0">
   <description>Prepare a dataset for the Velvet velvetg Assembler</description>
-  <version_command>velveth 2&gt;&amp;1 | grep "Version" | sed -e 's/Version //'</version_command>
   <macros>
     <import>macros.xml</import>
   </macros>
   <expand macro="requirements"/>
   <expand macro="stdio"/>
-  <command interpreter="python">
-    velveth_wrapper.py
-           '$out_file1' '$out_file1.extra_files_path'
-           $hash_length
-           $strand_specific
-           #for $i in $inputs
-                ${i.file_format}
-                ${i.read_type}
-                ${i.input}
-           #end for
-  </command>
+  <version_command>velveth 2&gt;&amp;1 | grep "Version" | sed -e 's/Version //'</version_command>
+  <command><![CDATA[
+    export OMP_NUM_THREADS="\${GALAXY_SLOTS:-1}" &&
+    export OMP_THREAD_LIMIT="\${GALAXY_SLOTS:-1}" &&
+    velveth
+      '$out_file1.extra_files_path'
+      $hash_length
+      #for $i in $inputs
+          ${i.file_format}
+          ${i.read_type}
+          #if str( $i.input_type.input_type_selector ) == "paireds":
+              -separate $i.input1 $i.input2
```

Comment:
single quotes for all data and text params.

Reply:
done